Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBus.h

```diff
@@ -49,6 +49,7 @@ namespace AZ::SceneAPI
         using ManifestUpdates = AZStd::vector<AZStd::shared_ptr<DataTypes::IManifestObject>>;
 
         virtual AZStd::optional<ManifestUpdates> GeneratePrefabGroupManifestUpdates(const Scene& scene) const = 0;
+        virtual AZStd::vector<AZStd::shared_ptr<DataTypes::IManifestObject>> GeneratePrefabMeshGroups(const Scene& scene) const = 0;
```

Comment:
I wonder is this should be called "Generate**Default**PrefabMeshGroups" since it is going to make all those default_XYZ mesh group names

Reply:
Sure no problem, done